test/blackbox-tests/test-cases/pkg/opam-repository-download.t

```diff
@@ -35,9 +35,8 @@ Make a mock repo tarball that will get used by dune to download the package
 Our custom cache folder should be populated with the unpacked tarball
 containing the repository:
 
-  $ find fake-xdg-cache | grep HEAD
-  fake-xdg-cache/dune/git-repo/HEAD
-  fake-xdg-cache/dune/git-repo/FETCH_HEAD
+  $ [ -f fake-xdg-cache/dune/git-repo/HEAD ]
+  > [ -f fake-xdg-cache/dune/git-repo/FETCH_HEAD ]
```

Comment:
Shouldn't there be `&&` to make sure the last check isn't ignored?

Reply:
I suppose I could change the `>` to a `$` if you are worried that they will intefere.

Reply:
> It currently should be equivalent to ; rather than &&.

That's exactly the problem. Add the following line to your test before the check:
```
$ rm fake-xdg-cache/dune/git-repo/HEAD
```
And note that your test is still passing.